### ISSUE TITLE
AI Speech - Not able to get it working

### ISSUE THREAD

**Issue by ruckupvr** [TheGameCreators/GameGuruRepo#3555]
Good day
I have used all the Scripts in Max to try and get the characters to speak and no luck. I tried stock WAV files and tried loading my own, nothing works. for example
"get close and speak" - When I get close nothing happens
"patrol and speak" - Character just walks past
"Stand and speak" nothing happens
I tried with about 10-12 different audio tracks including ones like Female hurt which works in game (kind of) but does not work in the speech scripts

**Comment by Bullyblasta** [comment]
i have the same issue. i have no clue.

**Comment by LeeBamberTGC** [comment]
@ruckupvr @Bullyblasta Just did a quick test with the latest version, the get close and speech, with some of my recorded voice worked fine:

https://user-images.githubusercontent.com/35840583/183685817-d0607a02-6346-4f85-90e5-a3b3dd33407a.mp4

Can you perhaps do a video showing dropping in a character, setting the behavior, recording some voice then running test level so I can compare what I saw with your results, thanks!

**Comment by megold16** [comment]
@LeeBamberTGC I'm not the original poster, but I have also been having the very same issue with speech behaviors for a few months now. I recorded the video below of me dropping in a fresh character, making him an Ally, setting the behavior, and recording the new voiceover. You'll notice that when I load into the game, the behavior doesn't work.

https://user-images.githubusercontent.com/111405972/186725785-7c4442d0-fd9c-4ce3-bef9-d7d3b95664f7.mp4

Help? 🤞

**Comment by synchromesh62** [comment]
@LeeBamberTGC 
Works but must be left as enemy ..
The OP set to ally which will not work.
Its always been that way for me which i assumed was default that it all went through the enemy behaviour. ?
Or is that a bug.

**Comment by LeeBamberTGC** [comment]
@megold16 Sorry for the late reply, just going through my backlog and taking a small breath away from DX12 conversion work. REALLY appreciate the video, a GREAT way to report bugs!!  I see the future :)  As Syncro says, a workaround does exist but definitely a bug and I have assigned to Zak as it should be a simple script tweak.

**Comment by Bullyblasta** [comment]
I appreciate it. Right now its just on my previous projects with the issue. I notices they worked fine when starting a new project but it was just slightly irritating after I turned them into smart objects in those projects. But I know theres not many of you and I want you guys to take your time and get GGMAX as perfect as possible and I love what u guys are doing and the direction your taking it keep up the good work guys. Sent from Mail for Windows From: Lee BamberSent: Saturday, October 29, 2022 12:34 PMTo: TheGameCreators/GameGuruRepoCc: Bullyblasta; MentionSubject: Re: [TheGameCreators/GameGuruRepo] AI Speech - Not able to get it working (Issue ***@***.*** Sorry for the late reply, just going through my backlog and taking a small breath away from DX12 conversion work. REALLY appreciate the video, a GREAT way to report bugs!! I see the future :) As Syncro says, a workaround does exist but definitely a bug and I have assigned to Zak as it should be a simple script tweak.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
